### ISSUE TITLE
Error: RevokeFollow / Unfollow

### ISSUE THREAD

**Issue by AndyHee** [friendica/friendica#11450]
### Bug Description

I get a few error messages like this in the Friendica log. The queue is very low though.

```
TypeError: Argument 1 passed to Friendica\Worker\Contact\RevokeFollow::execute() must be of the type int, null given in /var/www/friendica/src/Worker/Contact/RevokeFollow.php:39
Stack trace:
#0 [internal function]: Friendica\Worker\Contact\RevokeFollow::execute()
#1 /var/www/friendica/src/Core/Worker.php(497): call_user_func_array()
#2 /var/www/friendica/src/Core/Worker.php(387): Friendica\Core\Worker::execFunction()
#3 /var/www/friendica/src/Core/Worker.php(112): Friendica\Core\Worker::execute()
#4 /var/www/friendica/bin/worker.php(86): Friendica\Core\Worker::processQueue()
#5 {main}



TypeError: Argument 1 passed to Friendica\Worker\Contact\Unfollow::execute() must be of the type int, null given in /var/www/friendica/src/Worker/Contact/Unfollow.php:40
Stack trace:
#0 [internal function]: Friendica\Worker\Contact\Unfollow::execute()
#1 /var/www/friendica/src/Core/Worker.php(497): call_user_func_array()
#2 /var/www/friendica/src/Core/Worker.php(387): Friendica\Core\Worker::execFunction()
#3 /var/www/friendica/src/Core/Worker.php(112): Friendica\Core\Worker::execute()
#4 /var/www/friendica/bin/worker.php(86): Friendica\Core\Worker::processQueue()
#5 {main}

```

### Platform Info

Friendica Version:

Version 2022.03
Database version is 1452/1452

**Comment by MrPetovan** [comment]
Has this happened again? It looks like the RevokeFollow and Unfollow parameters were wrongly set when the worker task was created.

**Comment by MrPetovan** [comment]
Fixed by #11457
